### PR TITLE
GraphQL server in next.js lecture content

### PR DESCRIPTION
**Pull request by Eprince-hub**
This PR shows the complete code for the implementation of GraphQl server next.js and performing all mutation operations, authorization, and authentication

All changes made to reflect calendar content
Call animals from server component https://github.com/upleveled/graphql-example-winter-2023-vienna-austria/pull/3/commits/fa311dd3b67245b6f5b98b339822bf6caa5bf23c
Remove server actions https://github.com/upleveled/graphql-example-winter-2023-vienna-austria/pull/3/commits/8b7bba58e1669f1d7d19f41f3da698ee28b9bd49

### CODE REVIEW

File: util/ApolloClientProvider.tsx
Comment (karlhorky) [inline util/ApolloClientProvider.tsx:1]: let's switch this to be `app/ApolloClientProvider.tsx` - we keep components in the `app/` directory in our examples

File: util/client.ts
Comment (karlhorky) [inline util/client.ts:1]: - think of an alternative name for this file (remember, it should be identifiable in terms of the whole application)

Comment (Eprince-hub) [reply util/client.ts:1]: https://github.com/upleveled/graphql-example-winter-2023-vienna-austria/pull/3/commits/5f3fed305b8dc0a46c4705d2766b59658a49a878

File: util/ApolloClientProvider.tsx
Comment (Eprince-hub) [reply util/ApolloClientProvider.tsx:1]: https://github.com/upleveled/graphql-example-winter-2023-vienna-austria/pull/3/commits/af77cab2008ddb48ae71b58b5ef8f455256ba904

**Pull request merged**